std/hash/benchmark.zig

```diff
@@ -91,7 +86,7 @@ const Result = struct {
     throughput: u64,
 };
 
-const block_size: usize = 8192;
+const block_size: usize = 8 * 8192;
```

Comment:
I noticed that a block size of 8KiB was a fair bit lower than the usual L1 cpu cache so I've increased it. I noticed quite an improvement on one of the machines I tested this on.